pkg/synopsysctl/cmd_delete.go

```diff
@@ -40,75 +40,69 @@ var deleteCmd = &cobra.Command{
 
 // deleteBlackduckCmd deletes a Blackduck from the cluster
 var deleteBlackduckCmd = &cobra.Command{
-	Use:   "blackduck NAMESPACE",
-	Short: "Delete a Blackduck",
+	Use:   "blackduck NAME...",
+	Short: "Delete one or many Black Ducks",
 	Args: func(cmd *cobra.Command, args []string) error {
-		if len(args) != 1 {
-			return fmt.Errorf("this command takes 1 argument")
+		if len(args) <= 0 {
+			return fmt.Errorf("this command takes 1 or more arguments")
 		}
 		return nil
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
-		blackduckNamespace := args[0]
-		log.Infof("Deleting BlackDuck %s...", blackduckNamespace)
-
-		// Delete Blackduck with Client
-		err := blackduckClient.SynopsysV1().Blackducks(blackduckNamespace).Delete(blackduckNamespace, &metav1.DeleteOptions{})
-		if err != nil {
-			log.Errorf("error deleting the Blackduck: '%s'", err)
-			return nil
+		for _, blackduckNamespace := range args {
+			log.Infof("Deleting BlackDuck %s...", blackduckNamespace)
+			err := blackduckClient.SynopsysV1().Blackducks(blackduckNamespace).Delete(blackduckNamespace, &metav1.DeleteOptions{})
+			if err != nil {
+				log.Errorf("error deleting the Blackduck: '%s'", err)
```

Comment:
Provide the name of the blackduck with the error as well.  "error deleting black duck %s": %+v", name, err)